extensions/braze/braze-currents-users-event.schema.json

```diff
@@ -0,0 +1,300 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.braze.com/currents/users-event",
```

Comment:
In the below file, ,, look for airship and add another entry for braze
https://github.com/adobe/xdm/blob/master/meta.schema.json

Following string need to be enhanced to add braze
"pattern": "(https://ns\\.adobe\\.com/xdm/[a-z0-9-/]*)|(http://schema\\.org/.*)|(http://ns.adobe.com/adobecloud/core/1.0.*)|([http|https]://ns\\.adobe\\.com/adobecloudplatform/.*)|(https://tools\\.ietf\\.org/html/draft-kelly-json-hal-08/.*)|(http://www.iptc\\.org/.*)|(https://id3\\.org/id3v2.4/.*)|(https://ns\\.adobe\\.com/experience/[a-z0-9-/]*)|(https://ns\\.adobe\\.com/b2b/[a-z0-9-/]*)|(https://ns\\.airship\\.com/[a-z0-9-/]*)|(https://ns\\.braze\\.com/[a-z0-9-/]*)"